build.sh

```diff
@@ -32,19 +44,20 @@ popd
 # Copy rust build products
 mkdir -p deploy/rust
 cp version.md deploy/
-cp -R rust/build/${HOST_TRIPLE}/stage1/{bin,lib} deploy/rust/
-cp -R cargo/target/release/cargo deploy/rust/bin/
-rm -rf deploy/rust/lib/rustlib/src
+cp -R rust/build/${HOST_TRIPLE}/stage1/bin deploy/rust/
+mkdir -p deploy/rust/lib/rustlib/
+cp -R rust/build/${HOST_TRIPLE}/stage1/lib/rustlib/${HOST_TRIPLE} deploy/rust/lib/rustlib/
+cp -R rust/build/${HOST_TRIPLE}/stage1/lib/rustlib/bpfel-unknown-unknown deploy/rust/lib/rustlib/
+cp -R cargo/target/release/cargo${EXE_SUFFIX} deploy/rust/bin/
```

Comment:
shellcheck complain about this line "Double quote to prevent globbing and word splitting"

https://app.travis-ci.com/github/solana-labs/bpf-tools/builds/238558713#L203

Reply:
Done